src/Compilers/VisualBasic/Portable/VBResources.resx

```diff
@@ -1039,6 +1042,9 @@
   <data name="ERR_InvalidOverrideDueToReturn2" xml:space="preserve">
     <value>'{0}' cannot override '{1}' because they differ by their return types.</value>
   </data>
+  <data name="ERR_InvalidOverrideDueToTupleNames" xml:space="preserve">
+    <value>'{0}' cannot override '{1}' because they differ by their tuple element names.</value>
```

Comment:
Makes sense.

---
In reply to: [81813771](https://github.com/dotnet/roslyn/pull/14229#discussion_r81813771) [](ancestors = 81813771)